test/integrations/destinations/braze/dataDelivery/other.ts

```diff
@@ -0,0 +1,293 @@
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateProxyV1Payload } from '../../../testUtils';
+
+export const otherScenariosV1: ProxyV1TestData[] = [
+  {
+    id: 'braze_v1_other_scenario_1',
+    name: 'braze',
+    description:
+      '[Proxy v1 API] :: Scenario for testing Service Unavailable error from destination',
+    successCriteria: 'Should return 500 status code with error message',
+    scenario: 'Framework',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload({
+          endpoint: 'https://random_test_url/test_for_service_not_available',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            response: [
+              {
+                error:
+                  '{"error":{"message":"Service Unavailable","description":"The server is currently unable to handle the request due to temporary overloading or maintenance of the server. Please try again later."}}',
+                statusCode: 503,
+                metadata: {
+                  jobId: 1,
+                  attemptNum: 1,
+                  userId: 'default-userId',
+                  destinationId: 'default-destinationId',
+                  workspaceId: 'default-workspaceId',
+                  sourceId: 'default-sourceId',
+                  secret: {
+                    accessToken: 'default-accessToken',
+                  },
+                  dontBatch: false,
+                },
+              },
+            ],
+            statTags: {
```

Comment:
can we create a reusable function for this

Reply:
declared a common variable